src/DynamoCore/Graph/Nodes/VariableInputNode.cs

```diff
@@ -272,6 +272,28 @@ internal bool HandleModelEventCore(string eventName, UndoRedoRecorder recorder)
                     RemoveInputFromModel();
                     model.RegisterAllPorts();
                     return true; // Handled here.
+                case "SetInPortCount":
```

Comment:
Yeah, I need to learn about how to create Dynamo tests.
I mentioned that in the PR.